src/avt/DBAtts/MetaData/avtDatabaseMetaData.code

```diff
@@ -3440,6 +3445,23 @@ avtDatabaseMetaData::VarIsCompound(const std::string &v) const
     {
         return false;
     }
+
+    //
+    // Let's check if this is truly a known mesh.
+    //
+    std::string meshName = v.substr(beg + 1, end - beg - 1);
+
+    int nmeshes = GetNumMeshes();
+    for (int i = 0 ; i < nmeshes ; i++)
+    {
+        if (VariableNamesEqual(GetMeshes(i).name, meshName) ||
+            VariableNamesEqual(GetMeshes(i).blockTitle, meshName) ||
```

Comment:
It *should* match a mesh name only. The blockTitle and groupTitle are not needed here. In fact, those would match the *category* part.